src/book/book.rs

```diff
@@ -128,16 +128,24 @@ where
 pub enum BookItem {
     /// A nested chapter.
     Chapter(Chapter),
+    /// A nested virtual chapter.
+    VirtualChapter(VirtualChapter),
     /// A section separator.
     Separator,
```

Comment:
While we're making breaking changes, it may be a good idea to add a variant:
```rust
// To make sure clients have a `_ =>` case
#[doc(hidden)]
__NonExhaustive,
```
to ensure client libraries can handle new enum variants.

Reply:
Good idea. I will do that!